control/autoware_joy_controller/README.md

```diff
@@ -0,0 +1,43 @@
+# autoware_joy_controller
+
+## Role
+
+`autoware_joy_controller` operate a vehicle by a joy controller via Autoware (e.g. steering wheel, shift, turn signal, engage) by several kinds of joy sticks.
```

Comment:
~  is the packages to convert joy msg to autoware command (e.g. ~~ )for a vehicle ?

Reply:
https://github.com/tier4/AutowareArchitectureProposal.iv/pull/593/commits/1842070d0c115ba618c145e23c14b7747e19c9a0
Did it. Thanks